helpers/focus.js

```diff
@@ -13,6 +13,18 @@ const focusableElements = {
 	textarea: true
 };
 
+export function forceFocusVisible(elem) {
+	if (!isFocusable(elem)) {
```

Comment:
This handles the case where the element is something like a `d2l-button` where on its own it isn't focusable (but has a `focus()` method), so we need to find the real `button` element inside.